### PR TITLE
Origin/iac 591 vro backup

### PR DESCRIPTION
**Pull request by imihov1988**
### Checklist

- [x] Lint and unit tests pass locally with my changes
- [x] I have added relevant error handling and logging messages to help troubleshooting
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)
- [x] I have updated CHANGELOG.md with a short summary of the changes introduced
- [x] I have tested against live environment, if applicable
- [x] I have added relevant usage information (As-built)
- [x] Any structure and/or content vRA-NG improvements are synchronized with vra-ng and ts-vra-ng archetypes
- [x] Every new or updated Installer property is documented in docs/archive/doc/markdown/use-bundle-installer.md
- [x] Dependencies in pom.xml are up-to-date
- [x] My changes have been rebased and squashed to the minimal number of relevant commits
- [x] My changes have a descriptive commit message with a short title, including a `Fixed #XXX -` or `Closed #XXX -` prefix to auto-close the issue

### CODE REVIEW

**Review by ivo-kotev — CHANGES_REQUESTED** [review]
Please check the comments, they are very small and mostly related to the documentation and the  installer question, let's address them and we are ready to merge.

File: package-installer/src/main/java/com/vmware/pscoe/iac/installer/Installer.java [hunk 1]
```diff
@@ -655,7 +663,7 @@ public static void main(String[] args) throws ConfigurationException {
 
         if (input.allTrue(Option.SSH_IMPORT)) {
             PackageStoreFactory.getInstance(ConfigurationSsh.fromProperties(input.getMappings(ConfigurationPrefix.SSH.getValue())))
-                    .importAllPackages(getFilesystemPackages(PackageType.BASIC), false);
+                    .importAllPackages(getFilesystemPackages(PackageType.BASIC), false, vroEnableBackup);
```
[the remark this answers is not in the document]
Comment (imihov1988) [reply package-installer/src/main/java/com/vmware/pscoe/iac/installer/Installer.java:666]: On line 570 I set the variable vroEnableBackup to false after using the variable in the vRO logic, decided better to pass it with a variable.

File: package-installer/src/main/java/com/vmware/pscoe/iac/installer/Installer.java [hunk 2]
```diff
@@ -784,6 +792,8 @@ private static void loadInputInteractiveMode(Input input) {
 		if(!getFilesystemPackages(PackageType.VRO).isEmpty()){
             userInput(input, Option.VRO_IMPORT, "Import vRO packages?", true);
             if (input.anyTrue(Option.VRO_IMPORT)) {
+				userInput(input, Option.VRO_ENABLE_BACKUP, "Enable vRO backup?", true);
```
[the remark this answers is not in the document]
Comment (imihov1988) [reply package-installer/src/main/java/com/vmware/pscoe/iac/installer/Installer.java]: Addressed

File: docs/versions/latest/Release.md [hunk 3]
```diff
@@ -23,6 +23,7 @@
 [//]: # (Describe the feature)
 [//]: # (Optional But higlhy recommended Specify *NONE* if missing)
 [//]: # (#### Relevant Documentation:)
+[//]: # (### Implemented vRO backup during packages import)
```
[the remark this answers is not in the document]
Comment (imihov1988) [reply docs/versions/latest/Release.md]: Addressed

**Pull request merged**